docs/user/editor/patterns.md

```diff
@@ -129,10 +129,12 @@ Once editing a pattern shape, there are a few behaviors you should be aware of:
 - **Cancel** - Cancel edits and leave editing mode.
 
 ### Edit settings
-- **Snap to streets** - snaps segments between handles and stops to streets. If unchecked, segments will form straight lines between handles and stops (e.g., for drawing shapes for rail lines).
+- **Snap to streets** - snaps segments between handles and stops to streets. Can toggle the 'Avoid highways in routing' checkbox as needed.
+- **Snap to rail** - snaps segments between handles and stops to rail.
+- **None** - segments will form straight lines between handles and stops.
```

Comment:
Can you mention the dropdown selector and put these three snapping modes under it, please?